liveblog.php

```diff
@@ -541,30 +672,34 @@ public static function ajax_single_entry() {
 			break;
 		}
 
-		if ( ! $entries ) {
-			do_action( 'liveblog_entry_request_empty' );
-
-			self::json_return( array( 'entries' => array() ) );
-		}
-
 		$entries_for_json = array();
 
-		// Set up an array containing the JSON data for all Liveblog entries.
+		// Set up an array containing the JSON data for Liveblog entry.
 		foreach ( $entries as $entry ) {
 			$entries_for_json[] = $entry->for_json();
 		}
 
 		// Set up the data to be returned via JSON.
 		$result_for_json = array(
-			'entries'           => $entries_for_json,
-			'index'             => (int) filter_input( INPUT_GET, 'index' ),
-			'nextTimestamp'     => $next_timestamp,
-			'previousTimestamp' => $previous_timestamp,
+			'entries' => $entries_for_json,
 		);
 
-		do_action( 'liveblog_entry_request', $result_for_json );
+		if ( ! empty( $entries_for_json ) ) {
+			// Entries found
+			$result_for_json['index']             = (int) filter_input( INPUT_GET, 'index' );
+			$result_for_json['nextTimestamp']     = $next_timestamp;
+			$result_for_json['previousTimestamp'] = $previous_timestamp;
 
-		self::json_return( $result_for_json );
+			//self::json_return( $result_for_json );
```

Comment:
Code that isn't used or needed should be removed completely, rather than commenting it out. If it's needed in the future it can be restored via the Git history.